console/web3ext/web3ext.go

```diff
@@ -269,6 +269,16 @@ web3._extend({
 			call: 'admin_sleepBlocks',
 			params: 2
 		}),
+		new web3._extend.Method({
+			name: 'startHTTP',
+			call: 'admin_startHTTP',
+			params: 5,
+			inputFormatter: [null, null, null, null, null]
+		}),
+		new web3._extend.Method({
```

Comment:
@hyochan-brown Is this method avaliable? The change does not  show the method `stopHTTP`.